.github/workflows/ci.yml

```diff
@@ -183,6 +191,7 @@ jobs:
 
       - name: "Combine & check coverage"
         run: |
+          source .venv/bin/activate
```

Comment:
I wanted to note that `python -m build` can be updated to `uv run -m build`, which avoids activating the virtual environment. I'm not saying we should change it here, though.